src/controllers/version/downloads.rs

```diff
@@ -45,42 +58,6 @@ pub fn download(req: &mut dyn RequestExt) -> EndpointResult {
     }
 }
 
-/// Increment the download counts for a given crate version.
-///
-/// Returns the crate name as stored in the database, or an error if we could
-/// not load the version ID from the database.
-///
-/// This ignores any errors that occur updating the download count. Failure is
-/// expected if the application is in read only mode, or for API-only mirrors.
-/// Even if failure occurs for unexpected reasons, we would rather have `cargo
-/// build` succeed and not count the download than break people's builds.
-fn increment_download_counts(
```

Comment:
We may want to temporarily leave the current implementation available, behind an environment variable. That way, if we run into anything unexpected in production, we can easily switch back.

(We can always rollback a deploy, so this is more about making sure `master` is always deployable. If we do run into an issue, we can always add this back behind a flag, so I don't see this as a blocking concern.)

Reply:
Yeah, for now I'd merge it to master and rollback the deploy + revert the PR in case something goes wrong.